Parent/Parent/Course/CourseDetailsViewController.swift

```diff
@@ -120,7 +141,14 @@ extension CourseDetailsViewController: HorizontalPagedMenuDelegate {
         case .grades:
             return NSLocalizedString("Grades", comment: "")
         case .syllabus:
-            return NSLocalizedString("Syllabus", comment: "")
+            switch courses.first?.defaultView {
+            case .wiki:
+                return NSLocalizedString("Homepage", comment: "")
```

Comment:
Web calls this "Home". Did Dave want it to be "Homepage"?

Reply:
Peyton told me to call it Homepage